spec/features/menu_items/top_menu_item_spec.rb

```diff
@@ -120,11 +126,12 @@ def click_link_in_open_menu(title)
 
     context 'as a regular user' do
       it 'only displays projects, activity and news' do
+        skip "Figure out why the news item is not shown here any more"
```

Comment:
This will let you know once it starts showing up again
```suggestion
        pending "Figure out why the news item is not shown here any more"
```